ansible/roles/selenium-grid/files/selenium-grid.datadog.py

```diff
@@ -1,47 +0,0 @@
-
-
-import time
```

Comment:
I think we can keep this part. Telegraf listens on the statsd port and accepts datadog formatted stats so we keep using their python library in our stats scripts as there's no other well supported lib that does it right.